modules/ROOT/pages/anypoint-platform-cli.adoc

```diff
@@ -17,17 +17,10 @@ The CLI supports both interactive shell and standard CLI modes, and works with:
 
 == Prerequisites
 
-Anypoint-CLI is a NodeJS application. To install Anypoint Platform CLI on your computer you need to make sure that you have the NodeJS runtime environment installed on your computer, along with NPM, the default NodeJS package manager.
-Follow the NodeJS documentation to get both resources.
-
-Additionally, you  also need to install the git version management system on your computer. Installation instructions and links to downloads for your particular Operating System are available at the Git Documentation site.
-
-After you have installed NodeJS (with npm) and git, you can install Anypoint Platform CLI by opening a command-line interface and typing:
-
-[source]
-----
-> anypoint-cli 
-----
+* NodeJS, and NPM (Node's package manager) installed in your machine. +
+Go to the https://nodejs.org/en/download/[NodeJS download page] to get both resources.
+* Git version management system on your computer. +
+Go to the https://git-scm.com/downloads[Git Download site] to get this resource.
```

Comment:
lowercase Download for git

Reply:
Thank you! Good point. I changed it: b3758f0